src/theme/components/customTable.ts

```diff
@@ -1,13 +1,18 @@
+import { useColorModeValue } from "@chakra-ui/react";
+
 interface BaseStyleProps {
   columnHeadersBackgroundColor?: string;
   columnHeadersTextColor?: string;
   showRowDividers?: boolean;
   useRowHeaders?: boolean;
 }
 
-const cellBorderColorStyles = {
-  base: "ui.gray.light-cool",
-  md: "ui.gray.medium",
+const CellBorderColorStyles = () => {
```

Comment:
This doesn't need to be capitalized.

Reply:
I originally tried without the initial cap, but when I do that, I get the following error:
```
React Hook "useColorModeValue" is called in function "cellBorderColorStyles" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter.eslint[react-hooks/rules-of-hooks](https://reactjs.org/docs/hooks-rules.html)
```

Are there other ways to resolve this?